README.md

```diff
@@ -6,25 +6,27 @@ It is assumed that the environment already has these installed:
 * [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install versioned ruby
 * [ruby](https://www.ruby-lang.org/en/) currently using 2.4
 * [jekyll](https://jekyllrb.com/) currently using 3.5.2
+* [Docker](https://docs.docker.com/docker-for-mac/install/) to standardize builds across all contributors' local machines
```

Comment:
We should also include `docker-compose` in this list.  Again, you can refer to the README in the `bcda-app` repo [here](https://github.com/CMSgov/bcda-app/blob/master/README.md#dependencies).